### DISCUSSION TITLE
Speed throttling

### DISCUSSION THREAD

**Discussion by Xetaxheb** [UltimaHoarder/UltimaScraper#972] [Ideas]
Not a priority but if there's any way to limit so it doesn't use all available bandwidth that would be fantastic.

**Comment by UltimaHoarder** [comment]
I don't know how to do that

**Comment by drickerby** [comment]
I used a third party program on Windows called NetLimiter to limit the amount of bandwidth the python.exe process can use. It's working so far.

Bear in mind this is a paid program but it does give you a 28 day free trial.
